doc/classes/@GlobalScope.xml

```diff
@@ -457,6 +457,7 @@
 				    rotation = lerp_angle(min_angle, max_angle, elapsed)
 				    elapsed += delta
 				[/codeblock]
+				[b]Note:[/b] This method lerps through the shortest path between [code]from[/code] and [code]to[/code]. However, when these two angles are approximately [constant @GDScript.PI] apart, it's not obvious which way they lerp due to floating-point precision errors. For example, [code]lerp_angle(0, PI, weight)[/code] lerps counter-clockwise, while [code]lerp_angle(13 * PI, 14 * PI, weight)[/code] lerps clockwise.
```

Comment:
>when these two angles are approximately [constant @GDScript.PI] apart

To be precise: if angles are `PI + k * TAU` apart for any integer `k`. For example `0` and `99 * PI` angles are also kinda `PI` apart.

```
i=0  COUNTER-CLOCKWISE    lerp_angle(0, PI + 0 * TAU, 0.5) = -1.570796
i=1  COUNTER-CLOCKWISE    lerp_angle(0, PI + 1 * TAU, 0.5) = -1.570796
i=2  COUNTER-CLOCKWISE    lerp_angle(0, PI + 2 * TAU, 0.5) = -1.570796
i=3  COUNTER-CLOCKWISE    lerp_angle(0, PI + 3 * TAU, 0.5) = -1.570796
i=4  COUNTER-CLOCKWISE    lerp_angle(0, PI + 4 * TAU, 0.5) = -1.570796
i=5  CLOCKWISE            lerp_angle(0, PI + 5 * TAU, 0.5) =  1.570796
i=6  COUNTER-CLOCKWISE    lerp_angle(0, PI + 6 * TAU, 0.5) = -1.570796
i=7  CLOCKWISE            lerp_angle(0, PI + 7 * TAU, 0.5) =  1.570796
i=8  COUNTER-CLOCKWISE    lerp_angle(0, PI + 8 * TAU, 0.5) = -1.570796
i=9  CLOCKWISE            lerp_angle(0, PI + 9 * TAU, 0.5) =  1.570796
```